server/NucleiClassification/NucleiClassification.py

```diff
@@ -0,0 +1,181 @@
+        A list of n RGB colors
+
+    """
+
+    np.random.seed(seed)
+
+    color_list = []
+
+    h = np.random.random(n)
+    golden_ratio_conjugate = 0.618033988749895
+
+    for i in range(n):
+
+        h += golden_ratio_conjugate
+        h %= 1
+
+        color_list.append(colorsys.hsv_to_rgb(h, 1.0, 1.0))
+
+    np.random.seed(None)
+
+    return color_list
+
+
+def read_feature_file(args):
+
+    fname, feature_file_format = os.path.splitext(args.inputFeatureFile)
+
+    if feature_file_format == '.csv':
+
+        ddf = dd.read_csv(args.inputFeatureFile)
+
+    elif feature_file_format == '.h5':
+
+        ddf = dd.read_hdf(args.inputModelFile, 'Features')
+
+    else:
+        raise ValueError('Extension of output feature file must be .csv or .h5')
+
+    return ddf
+
+
+def check_args(args):
+
+    if not os.path.isfile(args.inputImageFile):
+        raise IOError('Input image file does not exist.')
+
+    if not os.path.isfile(args.inputModelFile):
+        raise IOError('Input model file does not exist.')
+
+
+def main(args):
+
+    print('\n>> CLI Parameters ...\n')
+
+    print args
+
+    #
+    # Initiate Dask client
+    #
+    print('\n>> Creating Dask client ...\n')
+
+    c = cli_utils.create_dask_client(args)
+
+    print c
+
+    #
+    # read model file
+    #
+    clf_model = joblib.load(args.inputModelFile)
+
+    #
+    # read feature file
+    #
+    ddf = read_feature_file(args)
+
+    if len(ddf.columns) != clf_model.n_features_:
+
+        raise ValueError('The number of features of the classification model '
+                         'and the input feature file do not match.')
+
+    #
+    # read nuclei annotation file
+    #
+    with open(args.inputNucleiAnnotationFile) as f:
+
+        nuclei_annot_list = json.load(f)['elements']
+
+    if len(nuclei_annot_list) != len(ddf.index):
+
+        raise ValueError('The number of nuclei in the feature file and the '
+                         'annotation file do not match')
+
+    #
+    # Perform nuclei classification
+    #
+    def predict_nuclei_class_prob(df, clf_model):
+
+        return pd.Dataframe(data=clf_model.predict_proba(df.as_matrix()),
+                            columns=clf_model.classes_)
+
+    outfmt = pd.DataFrame(columns=clf_model.classes_, dtype=np.float64)
+
+    df_class_prob = ddf.map_partitions(predict_nuclei_class_prob, clf_model,
+                                       meta=outfmt).compute()
+
+    pred_class = df_class_prob.idxmax(axis=1)
+
+    #
+    # Group nuclei annotations by class
+    #
+    num_classes = len(clf_model.classes_)
+
+    nuclei_annot_by_class = {c: [] for c in clf_model.classes_}
+
+    class_color_map = dict(zip(clf_model.classes_,
+                               gen_distinct_rgb_colors(num_classes, seed=1)))
+
+    for i in range(len(nuclei_annot_list)):
+
+        cur_class = pred_class.iloc[i]
+
+        cur_anot = nuclei_annot_list[i]
+        cur_anot['line_color'] = class_color_map[cur_class]
```

Comment:
According to https://github.com/girder/large_image/blob/master/docs/annotations.md, the attribute name is `"lineColor"` (camel-case).

According to the same document, colors must a string in one of several different formats. `rgb(123, 123, 123)` might be the easiest.